src/realm/object-store/shared_realm.cpp

```diff
@@ -1002,25 +1002,29 @@ void Realm::write_copy(StringData path, BinaryData key)
     }
 }
 
-void Realm::write_copy(const Config& config)
+void Realm::export_to(const Config& config)
 {
     std::string new_location = config.path;
     BinaryData encryption_key(config.encryption_key.data(), config.encryption_key.size());
-    if (config.sync_config) {
+    if (util::File::exists(new_location)) {
```

Comment:
Does this fail gracefully if the Realm exists and has an active sync session in another process?

Reply:
I don't know. It will work the same way as when you try to open the same synchronized realm from 2 different processes. Will that fail?

Reply:
I would expect something like "multiple sync agents detected" but yeah I guess this is not really different then the general case.